sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/DetectLanguageResult.java

```diff
@@ -18,9 +19,10 @@
      * @param textDocumentStatistics The text document statistics.
      * @param error The document error.
      * @param primaryLanguage The detected primary language.
+     * @param warnings A {@link IterableStream} of {@link TextAnalyticsWarning}.
```

Comment:
warnings should be in `DetectedLanguage` to be consistent with keyphrases. no?

Reply:
yes. Missed to revert this one back.